.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python.pythonPath": "C:\\Users\\Malactus\\AppData\\Local\\Programs\\Python\\Python39\\python.exe",
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/vircadia/Blender-Metaverse-Addon/pull/1/commits/283a74edae6ef56ef9f6bfa87314a17c31df98be